app/preprints/-components/submit/supplements/component.ts

```diff
@@ -11,24 +13,69 @@ interface SupplementsArgs {
     manager: PreprintStateMachine;
 }
 
+interface SupplementsForm{
+    node: Node;
+}
+
+const SupplementFormValidation: ValidationObject<SupplementsForm> = {
+    node: validatePresence({
```

Comment:
If this is an optional field, I don't think we even need a validator here. We may not even need a changeset on this page for that matter?